### ISSUE TITLE
dokanx.dll requires ProcMonDebugOutputx64(Win32).dll to run

### ISSUE THREAD

**Issue by don-Pardon** [BenjaminKim/dokanx#41]
After commit 49d90163a32afe680da121848e1dfbecdf10d0df dokanx.dll requires presence of ProcMonDebugOutputx64.dll or ProcMonDebugOutputWin32.dll (depending on platform). Could you please either add reference about this dll into installation manual or (if possible) make it optional in dokanx.dll.
Thanks!

**Comment by BenjaminKim** [comment]
@don-Pardon It means you can't run dokanx after the commit? Try to put `dokanx.dll` and `ProcMonDebugOutputXXX` at same directory.

**Comment by don-Pardon** [comment]
@BenjaminKim yeah, I did and it worked, but I think it's not reasonable to distribute `ProcMonDebugOutputXXX` alongside `dokanx.dll`. I suggest to use `ProcMonDebugOutputXXX` with debug version, and make it optional in release version of `dokanx.dll`.

**Comment by BenjaminKim** [comment]
@don-Pardon Understand. I totally agree that. I will try it. You can also help me it, though :)

**Comment by don-Pardon** [comment]
@BenjaminKim yeah, all right, I'll try today or tomorrow =)
